packages/ui/package.json

```diff
@@ -29,8 +29,8 @@
   ],
   "scripts": {
     "build:deps": "turbo build --filter=@open-condo/ui^...",
-    "build:styles": "find src/tokens -mindepth 1 -not -name 'tokens.json' -delete && node bin/prepare-tokens.js && node bin/buildStyleVars/runner.js --web less css",
```

Comment:
Find not working at the windows environment, right?)

Reply:
They have different args and flags there, yep

Reply:
ok then that's acceptable